Gemfile.lock

```diff
@@ -660,7 +660,7 @@ DEPENDENCIES
   webmock
 
 RUBY VERSION
-   ruby 3.2.2p53
+   ruby 3.3.0p0
 
 BUNDLED WITH
    2.4.22
```

Comment:
After installing 3.3.0, my local bundler version was also updated. I wonder if we should update the bundler version here as well? `bundle update --bundler`. Likely worth a separate commit, potentially a separate PR.

Reply:
I'll do it in a separate commit